ui/packages/markdown/src/Markdown.svelte

```diff
@@ -34,6 +34,8 @@
 
 	div :global(.math.inline svg) {
 		display: inline;
+		margin-bottom: 0.15em;
```

Comment:
margin-bottom: 0.25em looks better to me, seems like it hangs under the baseline at 0.15em